### DISCUSSION TITLE
[Question] How to use ActuatorNet Models for Manipualtor Arms and dextrous hands

### DISCUSSION THREAD

**Discussion by SakethTecholution** [isaac-sim/IsaacLab#1169] [Q&A] [answered]
### Question
@Mayankm96 @animesh-garg @mpgussert 
There is no documentation or demos provided for utilising the actuator net models of real world robot . Does this really improve or effect the training process and results in accurate motion . I have a dextrous hand but its very hard to tune or measure the physical parameters of hand to match simulation . In this case would actuatornet be helpful . If yes can you please guide me thorugh some of exisiting codes or docs to understand .

* A clear documentaiton is lacking in this part, It would be great if a sample demo with better documentation on real world robot control would be helpful

**Comment by garylvov** [comment]
Some examples of training actuator networks: 

https://github.com/Improbable-AI/walk-these-ways/tree/master/scripts/actuator_net
https://github.com/garylvov/scraps/tree/main/actuator_net

as for the process of training the network, I also found https://arxiv.org/pdf/1901.08652 helpful 

@Mayankm96 can provide better context on when to use an actuator network - from what I remember it's designed for non-linear motor dynamics and does not provide a benefit on DC motors but I'm not sure

**Comment by Dhoeller19** [comment]
Hi @SakethTecholution and thanks @garylvov for answering,
Indeed this actuator network is an expert topic that is described in the arxiv paper above. In the future,
we will release more documentation for sim-to-real transfer and we will describe the training process for the actuator networks more.

**Comment by SakethTecholution** [comment]
@Dhoeller19 @garylvov  Thanks for the reply. Just wanted to know will these be useful for manipulator arms . What do you suggest me to use implicit or actuators for system dynamics. I am using doosan cobot(a manipulator arm) with a dextrous hand attached to it .
Your suggestions onto it would be really helpful Thanks in advance..!

**Reply by glvov-bdai** [reply, marked as answer]
For the doosan cobot, I would recommend using an implicit actuator configuration. As for the hand, it's hard to tell without more information, but I would recommend starting with an implicit actuator configuration, and perhaps switching to a different configuration if you find poor sim to real transfer results

**Reply by SakethTecholution** [reply]
Thank you for the reply @glvov-bdai
